azure-iot-hub/tests/test_iothub_registry_manager.py

```diff
@@ -129,14 +130,98 @@ def mock_module_constructor(mocker):
 @pytest.fixture(scope="function")
 def mock_uamqp_send_message_to_device(mocker):
     mock_uamqp_send = mocker.patch.object(
-        IoTHubAmqpClientSharedAccessKeyAuth, "send_message_to_device"
+        azure.iot.hub.iothub_amqp_client.IoTHubAmqpClientSharedAccessKeyAuth,
+        "send_message_to_device",
     )
     return mock_uamqp_send
 
 
 @pytest.fixture
 def mock_uamqp_disconnect_sync(mocker):
-    return mocker.patch.object(IoTHubAmqpClientSharedAccessKeyAuth, "disconnect_sync")
+    return mocker.patch.object(
```

Comment:
Is this fixture even used?

Again, as usual, I realize this is kind of how the test module currently existed prior to you coming along, but I question when we start having to patch all these methods of an object (in all these fixtures). Perhaps the object itself could be patched (thus making it's methods all implicitly mocks as well)?

I dunno, just spitballing here, probably not worth upending the table. But do make sure that we don't have any dead code that isn't being used at all.